### PR TITLE
`asyncio.create_subprocess_shell()` -> `asyncio.create_subprocess_exec()`

### PR DESCRIPTION
**Pull request by altendky**
### Purpose:

avoid a ci hang in 3.11 on linux only.

### CODE REVIEW

**Comment by aqk** [comment]
I know dirname is where the binary is, but was dirname also the CWD of the previous code?

https://github.com/Chia-Network/chia-blockchain/pull/16944/files#diff-1054b958a0759914ed87f5b4bcdf6de32cecad90d3f380d79c1bad6ec4061f41R98

**Review by aqk — APPROVED** [review]

**Review by emlowe — APPROVED** [review]

File: chia/timelord/timelord_launcher.py [hunk 1]
```diff
@@ -88,8 +88,11 @@ async def spawn_process(
             dirname = path_to_vdf_client.parent
             basename = path_to_vdf_client.name
             resolved = await resolve(host, prefer_ipv6=prefer_ipv6)
-            proc = await asyncio.create_subprocess_shell(
-                f"{basename} {resolved} {port} {counter}",
```
Comment (arvidn) [inline chia/timelord/timelord_launcher.py:92]: yikes! you better hope none of those strings have spaces in them!

**Review by arvidn — APPROVED** [review]

**Comment by arvidn** [comment]
feel free to resolve my comment. I got carried away, it should have been a non-blocking comment

File: chia/timelord/timelord_launcher.py (hunk 1 above)
Comment (altendky) [reply chia/timelord/timelord_launcher.py:92]: Quite so...  All the more reason to not shove a shell in the middle.

**Pull request merged**